README.md

```diff
@@ -132,18 +132,19 @@ see [RELEASING.md](RELEASING.md).
 A typical installation from the release binaries might look like the following:
 
 ```shell script
-export WASI_VERSION=20
+export WASI_ARCH=x86_64
+export WASI_VERSION=24
 export WASI_VERSION_FULL=${WASI_VERSION}.0
```

Comment:
Could you remove the `export` keywords here while are you are it?  There should not be needed here or below.

Reply:
Do you mean `export`s for both URL construction and `WASI_SDK_PATH`? I thought that the second one is important

Reply:
As far as I can tell `export` is not needed in any of these cases.  The variables are used only within the script, and not by any sub-processes.